src/view.js

```diff
@@ -271,15 +271,42 @@ Marionette.View = Backbone.View.extend({
   // import the `triggerMethod` to trigger events with corresponding
   // methods if the method exists
   triggerMethod: function() {
+    var ret = Marionette._triggerMethod(this, arguments);
+
+    this._triggerEventOnBehaviors(arguments);
+    this._triggerEventOnParentLayout(arguments[0], _.rest(arguments));
+
+    return ret;
+  },
+
+  _triggerEventOnBehaviors: function(args) {
     var triggerMethod = Marionette._triggerMethod;
-    var ret = triggerMethod(this, arguments);
     var behaviors = this._behaviors;
     // Use good ol' for as this is a very hot function
     for (var i = 0, length = behaviors && behaviors.length; i < length; i++) {
-      triggerMethod(behaviors[i], arguments);
+      triggerMethod(behaviors[i], args);
     }
+  },
 
-    return ret;
+  _triggerEventOnParentLayout: function(eventName, args) {
```

Comment:
In a future version of Mn I think I'd prefer to only use https://github.com/marionettejs/backbone.marionette/pull/2818 and remove the auto proxying of events.  we throw sooooo many events with no listeners and the auto proxying cause edge case bugs where combining collectionview proxying and layout child proxying doubles events.